common/source/docs/common-electro-permanent-magnet-V3.rst

```diff
@@ -0,0 +1,143 @@
+.. _common-electro-permanent-magnet-gripper:
+
+=========================================
+Electro Permanent Magnet Gripper (OpenGrab EPM v3)
+=========================================
+
+This article shows how to use the `Electro Permanent Magnet Gripper (OpenGrab EPM v3) <http://nicadrone.com/index.php?id_product=66&controller=product>`__
+with ArduPilot.
+
+`Problems issues feedback <mailto:Andreas@NicaDrone.com?Subject=Problems%20issues%20feedback>`__
+
+
+Overview
+========
+
+The *OpenGrab EPM v3* (from NicaDrone.com) is a small (less than 4cm x 4cm x 2cm)
+Electro Permanent Magnet gripper capable of 20kg of force meaning it can
```

Comment:
kg are a unit of mass, not force. Please clarify what you're trying to say here. I would suggest perhaps "gripper capable of lifting about 15kg of cargo" would be useful.